apollo-federation/src/sources/connect/models/validation.rs

```diff
@@ -856,4 +890,22 @@ mod test_validate_source {
         assert_eq!(errors[1].code, Code::NoSourceImport);
         assert_snapshot!(errors[1].message, @r###"The `@source` directive is not imported. Try adding `@source` to `import` for `@link(url: "https://specs.apollo.dev/connect")`"###);
     }
+
```

Comment:
You'll need to rebase onto the PR I just merged - these tests are now implemented using the `glob` feature in insta.